javatests/google/registry/rde/imports/RdeDomainImportActionTest.java

```diff
@@ -126,6 +139,21 @@ public void test_mapreduceTwiceDoesNotDuplicateResources() throws Exception {
     checkHistoryEntry(historyEntries.get(0), domain);
   }
 
+  /** Ensures that dns publishing is kicked off on domain import */
+  @Test
+  public void test_mapreducePublishesToDns() throws Exception {
+    pushToGcs(DEPOSIT_1_DOMAIN);
+    runMapreduce();
+    DnsQueue dnsQueue = DnsQueue.create();
```

Comment:
This whole section from here to the end of the test is something we usually just use the assertDnsTasksEnqueued() helper for, see here:

https://github.com/google/nomulus/blob/b70f57b7c795ba60ecbce78607452fd5fa7b793e/javatests/google/registry/testing/TaskQueueHelper.java#L265

Takes care of a lot of the boilerplate.  I've already started importing this PR though so I'll just make the change on my end, no need to update here.